src/samples/cover-list/v1/first-page.json

```diff
@@ -88,6 +88,278 @@
+                    "mediaType": "image/jpeg",
+                    "uri": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif/full/full/0/default.jpg",
+                    "filename": "an-image.jpg"
+                },
+                "size": {
+                    "width": 4194,
+                    "height": 4714
+                }
+            },
+            "item": {
+                "status": "poa",
+                "id": "09560",
+                "version": 1,
+                "type": "research-article",
+                "doi": "10.7554/eLife.09560",
+                "authorLine": "Lee R Berger et al.",
+                "title": "<i>Homo naledi</i>, a new species of the genus <i>Homo</i> from the Dinaledi Chamber, South Africa",
+                "stage": "published",
+                "published": "2015-09-10T00:00:00Z",
+                "statusDate": "2015-09-10T00:00:00Z",
+                "volume": 4,
+                "elocationId": "e09560",
+                "pdf": "https://elifesciences.org/content/4/e09560.pdf",
+                "subjects": [
+                    {
+                        "id": "genomics-evolutionary-biology",
+                        "name": "Genomics and Evolutionary Biology"
+                    }
+                ],
+                "image": {
+                    "thumbnail": {
+                        "uri": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif",
+                        "alt": "",
+                        "source": {
+                            "mediaType": "image/jpeg",
+                            "uri": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif/full/full/0/default.jpg",
+                            "filename": "an-image.jpg"
+                        },
+                        "size": {
+                            "width": 4194,
+                            "height": 4714
+                        },
+                        "focalPoint": {
+                            "x": 25,
+                            "y": 75
+                        }
+                    }
+                }
+            }
+        },
+        {
+            "title": "<i>Infection, statistics and public health</i>",
+            "impactStatement": "What attracted you to studying epidemiology?",
+            "image": {
+                "uri": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif",
+                "alt": "",
+                "source": {
+                    "mediaType": "image/jpeg",
+                    "uri": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif/full/full/0/default.jpg",
+                    "filename": "an-image.jpg"
+                },
+                "size": {
+                    "width": 4194,
+                    "height": 4714
+                }
+            },
+            "item": {
+                "type": "interview",
+                "id": "2",
+                "interviewee": {
```

Comment:
The item only needs to have properties expected in a snippet. I can help with this.